server/graphql/core/src/lib.rs

```diff
@@ -13,8 +13,8 @@ use actix_web::web::Data;
 use actix_web::HttpRequest;
 use async_graphql::{Context, Request, Response};
 
+use actix_web::http::header::COOKIE;
```

Comment:
It seems like changing the reqwest import meant that we need to be correct about which `header` package we use through out the code base...